app/javascript/mastodon/features/getting_started/index.js

```diff
@@ -65,18 +66,22 @@ export default class GettingStarted extends ImmutablePureComponent {
       }
     }
 
+    if (!multiColumn || !columns.find(item => item.get('id') === 'DIRECT')) {
+      navItems.push(<ColumnLink key='4' icon='envelope' text={intl.formatMessage(messages.direct)} to='/timelines/direct' />);
+    }
+
```

Comment:
Why is it not grouped with the tests for other columns?

Reply:
Because it's doing the exact opposite test; since it doesn't show in the navigation bar in single column mode (space concerns, as I mentioned already), it needs to always show up there.

Reply:
Ok, makes sense!